### DISCUSSION TITLE
command event calling

### DISCUSSION THREAD

**Discussion by Osprey010** [jagrosh/MusicBot#1286] [Questions]
I was wondering when and where in the code you execute the command events. I am a little familiar with  / commands and thought that I could implement them in to the JMusic bot that I'm hosting. I have a little [Node.js project](https://github.com/Osprey010/Node-js-projekt)  that I did quite a while ago that I added to github today. It is a bot to send my ip to a discord server it recives a command. I never really got it working and thought it was time to archive it.

**Comment by MichailiK** [comment]
Commands are handled by the [JDA-Utilities](https://github.com/JDA-Applications/JDA-Utilities) library. Note that [this project’s README states that you won’t receive support with using/editing the code here.](https://github.com/jagrosh/MusicBot#editing)
